diff --git a/fobos/fobos.c b/fobos/fobos.c
index bf1a8d4..3d1af6a 100644
--- a/fobos/fobos.c
+++ b/fobos/fobos.c
@@ -174,7 +174,7 @@ int fobos_rx_get_api_info(char * lib_version, char * drv_version)
 {
     if (lib_version)
     {
-        strcpy(lib_version, LIB_VERSION" "__DATE__" "__TIME__);
+        strcpy(lib_version, LIB_VERSION" Mar 25 2025 13:30:05");
     }
     if (drv_version)
     {
